plugins/nf-float/src/main/com/memverge/nextflow/FloatConf.groovy

```diff
@@ -67,14 +72,27 @@ class FloatConf {
             return
         }
         username = node.username
+        if (StringUtils.length(username) == 0) {
+            username = System.getenv(MMC_USERNAME)
+        }
         password = node.password
+        if (StringUtils.length(password) == 0) {
+            password = System.getenv(MMC_PASSWORD)
+        }
         if (node.address instanceof Collection) {
             addresses = (node.address as Collection).collect {
                 it.toString()
             }
```

Comment:
```suggestion
            addresses = node.address*.toString()
```

Reply:
When you check the type with `instanceof` like this, you don't have to cast it

Reply:
I use your change but looks like it cannot pass the compile.
```
[Static type checking] - Spread operator can only be used on collection types
 @ line 77, column 29.
               addresses = node.address*.toString()
```

I updated it to something like this:
``` groovy
if (node.address instanceof Collection) {
            addresses = node.address.collect {it.toString()}
        }
```

Reply:
Fair enough, I guess the compiler wasn't as smart as I hoped